addons/fire/CfgSounds.hpp

```diff
@@ -0,0 +1,26 @@
+// weird ass concatenation syntax. PBO Project complains otherwise...
+#define CONCAT(a,b) a####b
```

Comment:
```suggestion
#define CONCAT(a,b) a##b
```
This is used elsewhere and works fine, what's the actual problem?

Reply:
Apparently PBO project complains/complained about it
https://github.com/acemod/ACE3/pull/7565#discussion_r391812344